internal/python/plan.go

```diff
@@ -83,7 +83,7 @@ func DeterminePackageManager(ctx *pythonPlanContext) types.PackageManager {
 		filename         string
 	}{
 		{types.PythonPackageManagerPipenv, "Pipfile"},
-		{types.PythonPackageManagerPoetry, "pyproject.toml"},
+		{types.PythonPackageManagerPdm, "pyproject.toml"},
```

Comment:
Don't rename this variable to a specific package manager! Instead, rename it to `PythonPackageManagerWithPyProject` or something like this.

Note that `setuptools` supports using `pyproject.toml` to describe a package. Therefore, it may still need to be installed with `pip` even if a project contains `pyproject.toml`. [details](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html)